src/Pages/Capture.elm

```diff
@@ -310,7 +312,7 @@ view model =
                 , if String.isEmpty model.error then
                     column [ width fill, height fill, spacing 50 ]
                         [ column [ centerX, spacing 10 ]
-                            [ EltInput.text [ EltInput.focusedOnLoad ]
+                            [ EltInput.text [ EltInput.focusedOnLoad, htmlAttribute <| id "capture-text" ]
```

Comment:
the function `focusOnLoad` from elm-ui will put the focus on the input on the first page load. However when the user navigate between pages this function is not recalled. Adding an id on the input and forcing the focus with `Task.attempt (\_ -> None) (Dom.focus "capture-text")` when the page initialised resolve this issue.